src/back-end/lib/resources/opportunity/sprint-with-us.ts

```diff
@@ -1625,6 +1642,101 @@ const update: crud.Update<
             });
           }
         }
+        case "submitIndividualQuestionEvaluations": {
+          for (const id of request.body.value.evaluations) {
```

Comment:
You could map over this instead since it's synchronous and each validation can be done independently.